faw/pdf-observatory/ui/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
```

Comment:
Figured I might as well update this; even though we don't invoke it, we might someday want to.

This command is destructive, like `vue-cli-service lint` was.